t/plugin/dynamic-upstream.t

```diff
@@ -322,10 +318,11 @@ passed
 
 
 
-=== TEST 7: send a request to empty match configuration
+-- Currently `nodes` is single domain and will print `502` error log.
+-- TODO: Error log printing is not allowed.
+=== TEST 9: node is domain name, the request normal
```

Comment:
we can add tip `TODO` to the title of test

Reply:
fixed